DigitalOffice/Tests/CompanyServiceTests/Controllers/CompanyControllerTests.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System;
+using LT.DigitalOffice.CompanyService.Commands.Interfaces;
+using LT.DigitalOffice.CompanyService.Controllers;
+using LT.DigitalOffice.CompanyService.Models;
+using Moq;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.CompanyServiceUnitTests.Controllers
+{
+    public class CheckRightsControllerTests
+    {
+        private CompanyController companyController;
+        private Mock<IAddCompanyCommand> addCompanyCommandMock;
+        private Guid companyId;
+
+        [SetUp]
+        public void SetUp()
+        {
+            companyController = new CompanyController();
+            addCompanyCommandMock = new Mock<IAddCompanyCommand>();
+        }
+
+        [Test]
+        public void ShouldReturnTheSameValueAsCommandWhenAddingCompany()
+        {
+            addCompanyCommandMock.Setup(command => command.Execute(It.IsAny<AddCompanyRequest>()))
+                .Returns(companyId)
+                .Verifiable();
+
+            Assert.That(companyController.AddCompany(addCompanyCommandMock.Object, It.IsAny<AddCompanyRequest>()),
+                Is.EqualTo(companyId));
+            addCompanyCommandMock.Verify();
+        }
+
+        [Test]
+        public void ShouldThrowExceptionWhenCommandThrowsException()
```

Comment:
You can clarify the description of the method.

Reply:
If the need arises, then the person adding new tests will be able to rename